### PR TITLE
Yet Another Gold Remake

### PR DESCRIPTION
**Pull request by Haganeko**
Could we put this in a test client and uh test it before we push it out?

### CODE REVIEW

**Comment by chrisinajar** [comment]
I'll merge right before release sunday so you can put more numbers in if you want :+1: 

will blindly merge, enjoy the experiments :D

**Comment by chrisinajar** [comment]
merge ETA 36 hours. Keep hacking away if you'd like :)

**Comment by Haganeko** [comment]
I'm trying to test it. Got a lobby open and all. Come over!

**Comment by Haganeko** [comment]
Apparently this wasn't the way to go.

**Comment by Haganeko** [comment]
Uh... We're back in business?

**Comment by Haganeko** [comment]
@chrisinajar halp

File: game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt [hunk 1]
```diff
@@ -126,7 +126,6 @@
       {
         "var_type"                                        "FIELD_INTEGER"
         "replenish_mana"                                  "200 300 475 700"
-      }
```
Comment (chrisinajar) [inline game/scripts/npc/items/farming/item_greater_guardian_greaves_4.txt]: ^-- this

**Pull request merged**